content/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork.md

```diff
@@ -27,6 +27,18 @@ topics:
 
 If the changes from the upstream repository cause conflicts, {% data variables.product.company_short %} will prompt you to create a pull request to resolve the conflicts.
 
+## Syncing a fork with the {% data variables.product.prodname_cli %}
+
+{% data variables.product.prodname_cli %} is an open source tool for using {% data variables.product.prodname_dotcom %} from your computer's command line. When you're working from the command line, you can use the {% data variables.product.prodname_cli %} to save time and avoid switching context.
```

Comment:
```suggestion
{% data reusables.cli.about-cli %} To learn more about {% data variables.product.prodname_cli %}, see "[About {% data variables.product.prodname_cli %}](/github-cli/github-cli/about-github-cli)."
```
Adding a link for people to learn more about GitHub CLI so that they can install it if they haven't done so before reading this article ⚡